.github/workflows/CI.yml

```diff
@@ -0,0 +1,46 @@
+name: CI
+on:
+  pull_request:
+  push:
```

Comment:
Would we want to setup a schedule to run testing everyday? Not sure who actively maintains these packages, and if they're interested in something like that. Just a thought.

Reply:
Probably not; anyone watching JuliaCollections would get a notification, and that's a lot of noise for packages that don't see a lot of changes.

Reply:
FYI in a few moments there are going to be a number of other mass PRs and notifications about public keys being added. That's all me setting up the modern workflow.